benchmark/worker/atomics-wait.js

```diff
@@ -7,6 +7,10 @@ const bench = common.createBenchmark(main, {
 });
 
 function main({ n }) {
+  if (typeof SharedArrayBuffer === 'undefined') {
+    throw new Error('SharedArrayBuffers must be enabled to run this benchmark');
+  }
```

Comment:
`main` is going to be executed more than once. As such, it's probably best to move the check to the top of the file.